src/main/java/com/example/gamelist/GamelistService.java

```diff
@@ -0,0 +1,32 @@
+package com.example.gamelist;
+
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+import static com.example.gamelist.GamelistService.UpdateRequest.name;
+
+@Service
+public class GamelistService {
```

Comment:
GamelistになっているところGameServiceに直しましょう。
```suggestion
public class GameService {
```

Reply:
<img width="433" alt="スクリーンショット 2023-12-26 16 07 35" src="https://github.com/otyacazyu/gamelist/assets/143002834/564cfe92-a6c5-4b3c-af9b-3eeec340bb69">
099fce73d93dd3f63d569fb4e12d08fc77192887

GameServiceに変更しました

Reply:
OKです。

Reply:
Resolveしましょう。